critiquebrainz/frontend/templates/index.html

```diff
@@ -35,11 +37,7 @@ <h3 style="margin-bottom:-5px;margin-top:0px;">{{ _('Popular reviews') }}</h3>
         <div class="col-md-6 review">
           <div class="cover-art-container pull-left">
             <a href="{{ url_for('review.entity', id=review.id) }}"><img /></a>
-            <script>
-              $("img").last()
-                  .on("error", function () { $(this).attr("src", "/static/img/missing-art.png"); })
-                  .attr("src", "//coverartarchive.org/release-group/{{ review.entity_id }}/front-250");
-            </script>
+            {{ entity_image_setter(review.entity_id, review.is_of_event) }}
```

Comment:
Can you make this macro more independent by making it add `<img />` tag as well? And a better name then would be just `cover_art`.